src/lib/stack.js

```diff
@@ -44,7 +48,7 @@ const current = function() {
  * @returns {Promise}
  */
 const init = function(options) {
-  stack = internal
+  stack = (options.cozyClient) ? client : internal
```

Comment:
I would find it clearer if it was written

`stack = (options.cozyClient) ? newStack : legacyStack`

Reply:
:+1: but it is temporary and disapear at the last stage of this PR anyways